docs/UserGuide.md

```diff
@@ -41,10 +41,10 @@ TAS can get your scholarship applicant management tasks done faster than traditi
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g `n/NAME [m/MAJOR]` can be used as `n/John Doe m/Buisness` or as `n/John Doe`.
```

Comment:
Business spelled wrongly

Reply:
ooops this one really my bad.My english really sigh...

Reply:
Ok thank you it has been resolved.